docs/getting-started.md

```diff
@@ -348,44 +567,82 @@ public static render(extensionUri: vscode.Uri) {
+Back in `HelloWorldPanel.ts` add the nonce to the webview `script` tag and create a content security policy `meta` tag so that only loading scripts with a nonce is allowed.
 
 ```typescript
 // file: src/panels/HelloWorldPanel.ts
 
+import { getNonce } from "../utilities/getNonce";
+
+// ... other code ...
+
 private _getWebviewContent(webview: vscode.Webview, extensionUri: vscode.Uri) {
-  // ... other code ...
+  const webviewUri = getUri(webview, extensionUri, ["out", "webview.js"]);
+
+  const nonce = getNonce();
 
   return /*html*/ `
     <!DOCTYPE html>
     <html lang="en">
       <head>
         <meta charset="UTF-8">
-        <meta name="viewport" content="width=device-width,initial-scale=1.0">
-        <script type="module" src="${toolkitUri}"></script>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="Content-Security-Policy" content="default-src 'none'; script-src 'nonce-${nonce}';">
         <title>Hello World!</title>
       </head>
       <body>
         <h1>Hello World!</h1>
         <vscode-button id="howdy">Howdy!</vscode-button>
+        <script type="module" nonce="${nonce}" src="${webviewUri}"></script>
       </body>
     </html>
   `;
 }
 ```
 
+_Note: As your extension grows you will likely want to add custom styles, fonts, and/or images to your webview. If you do, you will need to update the content security policy meta tag to explicity allow for these resources. This [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy) provides further information on the topic. Also here's a simple example of configuring these:_
+
+```js
+private _getWebviewContent(webview: vscode.Webview, extensionUri: vscode.Uri) {
+  return /*html*/ `
+    // ... other markup ...
+    <meta http-equiv="Content-Security-Policy" content="default-src 'none'; style-src ${webview.cspSource}; font-src ${webview.cspSource}; img-src ${webview.cspSource} https:; script-src 'nonce-${nonce}';">
+  `;
+}
+```
+
+### Make sure everything works
+
+It's time to run a test to see if everything works. Just as before, open the extension in the Extension Development Host window by pressing `F5` and then run the `Hello World` command.
+
 ![Testing That The Toolkit Works](./assets/images/toolkit-button-test.gif)
 
-Theming is built right into the components, so you can test that out, too. Open the Command Pallette (`Crtl + Shift + P` or `Cmd + Shift + P` on Mac), search for "Preferences: Color Theme," and cycle through all the themes to see the button change.
+Theming is built right into the components, so you can test that out too. Open the Command Palette (`Crtl + Shift + P` or `Cmd + Shift + P` on Mac), search for "Preferences: Color Theme," and cycle through all the themes to see the button change.
```

Comment:
nit: there is a chord to access this directly. Up to you if you want to reference this vs. the two step way via the command palette.
<img width="338" alt="CleanShot 2023-01-17 at 15 06 21@2x" src="https://user-images.githubusercontent.com/25163139/213031820-717a15c1-a4b9-4407-b1d4-53fa999a5fb2.png">

Reply:
Oh TIL. Definitely will add it!

Reply:
Actually, scratch that, I have a GIF that demos using the two-step method and I don't really feel like re-recording/re-editing so I'll leave it for now 😅